src/test/java/uk/gov/hmcts/reform/sscs/service/EvidenceShareServiceTest.java

```diff
@@ -73,6 +74,8 @@ public void setUp() {
         evidenceShareService = new EvidenceShareService(sscsCaseCallbackDeserializer, documentManagementServiceWrapper, documentRequestFactory,
             evidenceManagementService, bulkPrintService, evidenceShareConfig, ccdCaseService, idamService, roboticsHandler);
         when(evidenceShareConfig.getSubmitTypes()).thenReturn(Collections.singletonList("paper"));
+
+        ReflectionTestUtils.setField(evidenceShareService, "sendToDwpFeature", true);
```

Comment:
shouldn't this be set to true by default?

Reply:
Yeh but it was conflicting with other tests that had turned the flag off for some reason